OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -289,6 +303,7 @@ Action InitDisplayPanel(Widget panel)
 
 			uiScaleDropdown.IsDisabled = () => disableUIScale;
 
+			panel.Get("SCREEN_SELECTION").IsVisible = () => ds.Mode != WindowMode.Windowed;
```

Comment:
Style nit: Please rename this to `DISPLAY_SELECTION` to match `DISPLAY_SELECTION_DROPDOWN`